### DISCUSSION TITLE
Junction table, add extra data when selecting (ex: quantity in carts_products)

### DISCUSSION THREAD

**Discussion by development-vargamarcel** [directus/directus#12494] [Feature Requests]
Hello, really this is last priority feature but still would be nice.

Imagine this junction table (please note **quantity** field :
![Screenshot_2022-03-31-00-53-35-71_4641ebc0df1485bf6b47ebd018b5ee76](https://user-images.githubusercontent.com/58333260/160944191-3d93c454-1a9b-464e-ba34-424b893ca566.jpg)

This is how i can add or edit a cart:
![Screenshot_2022-03-31-00-57-18-06_4641ebc0df1485bf6b47ebd018b5ee76](https://user-images.githubusercontent.com/58333260/160944532-e7ebfd5c-6897-43c8-9742-abbe771034a9.jpg)

**When I add or edit a cart, I would like to not only be able to select products I want to put inside it, but also the quantity for each one and who knows other data too.**

**Comment by rijkvanzanten** [comment]
If you add additional fields to the junction field, they should already show up on the edit drawer when you create/edit an item in this interface:

<img width="1279" alt="CleanShot 2022-03-31 at 10 22 21@2x" src="https://user-images.githubusercontent.com/9141017/161078076-e1988e80-3840-461d-974f-b6c59fdc99d2.png">

**Reply by development-vargamarcel** [reply]
**You are right ,thank you very much!**
Didn't notice because i always added already existing 'products', and not creating new ones from this view.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
